### PR TITLE
javascript-functions-practice - Cynthia Huynh

### PR DESCRIPTION
**Pull request by HuynhCynthia**

### CODE REVIEW

**Comment by HuynhCynthia** [comment]
Screen Shot 2023-01-31 at 3 57 23 PM

File: javascript-functions-practice/get-character-at-index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* exported getCharacterAtIndex */
+function getCharacterAtIndex(string, index) {
+  var convertStringToArray = Array.from(string);
```
Comment (shawnkost) [inline javascript-functions-practice/get-character-at-index.js:3]: You don't need to actually convert this to an array. You can just do `string[index]`

File: javascript-functions-practice/get-first-character.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* exported getFirstCharacter */
+function getFirstCharacter(string) {
+  var convertStringToArray = Array.from(string);
```
Comment (shawnkost) [inline javascript-functions-practice/get-first-character.js:3]: Same note as above ^. You don't need to actually convert this to an array, you can just do `string[0]`

File: javascript-functions-practice/get-second-character.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/* exported getSecondCharacter */
+function getSecondCharacter(string) {
+  var convertStringToArray = Array.from(string);
```
Comment (shawnkost) [inline javascript-functions-practice/get-second-character.js:3]: Same note as above, you don't need to convert to an array here. You can just do `string[1]`

**Review by shawnkost — APPROVED** [review]
Approved, but I left a few notes below! Make sure to check them out! 👍🏻

**Pull request merged**